src/category/service.ts

```diff
@@ -3,31 +3,30 @@ import {
   Injectable,
   InternalServerErrorException,
 } from '@nestjs/common';
-import { Category } from '@prisma/client';
 import { PrismaService } from 'src/prisma/service';
-import { categoryDTO } from './dtos';
+import { CategoryDTO } from './dtos';
 
 @Injectable()
 export class CategoryService {
```

Comment:
Also, use plural noun.
```suggestion
export class CategoriesService {
```